test/unit/models/BrandingConfig.test.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable no-unused-expressions */
+const { expect } = require('chai');
+
+describe('BrandingConfig Model (semantic variables whitelist)', () => {
+  it('creates with empty variables map (defaults)', async () => {
+    const created = await BrandingConfig.create({ name: 'tenant-a', variables: {} }).fetch();
+    expect(created.variables).to.deep.equal({});
+    expect(created.version).to.equal(0);
+    expect(created.hash).to.equal('');
+  });
+
+  it('accepts whitelisted semantic variable keys', async () => {
+    const created = await BrandingConfig.create({
+      name: 'tenant-b',
+      variables: {
+        'site-branding-area-background-color': '#ffffff',
+        'panel-branding-color': '#000000',
+        'branding-font-family': 'Arial, sans-serif'
+      }
+    }).fetch();
+    expect(created.variables).to.have.keys([
+      'site-branding-area-background-color',
+      'panel-branding-color',
+      'branding-font-family'
+    ]);
+  });
+
+  it('rejects old raw color slot name (branding-color-N not allowed)', async () => {
+    let error;
+    try {
```

Comment:
Could this use the [chai `.throw`](https://www.chaijs.com/api/bdd/#method_throw) assertion instead?